.dependabot/config.yml

```diff
@@ -8,3 +8,12 @@ update_configs:
           dependency_type: all
           update_type: all
     version_requirement_updates: increase_versions
+
+  - package_manager: javascript
+    directory: /examples/
```

Comment:
I'm not super confident that Dependabot will find the example `package.json`s from this. We'll see after this is merged.